csvcubed/tests/unit/utils/test_json.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from tests.unit.test_baseunit import get_test_cases_dir
+from csvcubed.utils.json import load_json_document
+
+_json_test_cases_dir = get_test_cases_dir() / "utils" / "json"
+
+
+def test_loading_json_from_file_uri():
+    """
+    Ensure we can load a JSON document from the local file system with a `file://` URI.
+    """
+
+    document = load_json_document(
+        "file://" + str(_json_test_cases_dir / "json-document.json")
+    )
+    assert document.get("This") == "Is Valid JSON"
+
+
+def test_loading_json_from_file_path():
+    """
+    Ensure we can load a JSON document from the local file system with a `pathlib.Path`.
+    """
+
+    document = load_json_document(_json_test_cases_dir / "json-document.json")
+    assert document.get("This") == "Is Valid JSON"
+
+
+def test_loading_json_from_url():
+    """
+    Ensure we can load a JSON document from the local file system with a `pathlib.Path`.
+    """
+
+    document = load_json_document(
```

Comment:
I know I wrote this test, but could you use the [request mock stuff](https://github.com/GSS-Cogs/csvcubed/blob/main/csvcubed/tests/unit/readers/conftest.py#L9) I did elsewhere to ensure we don't *actually* have to make an HTTP request for this test?

Something like the following should be sufficient in this test:

```python
>>> with requests_mock.Mocker() as m:
...     m.get('http://test.com', text='data')
...     requests.get('http://test.com').text
...
'data'
```

Reply:
@robons No problem at all and I have sorted this now.